### ISSUE TITLE
Logits are the same as softmax scores

### ISSUE THREAD

**Issue by mihow** [RolnickLab/ami-data-companion#91]
The logits returned with classifications are a copy of the softmax scores! This was fixed in the `ood` branch but never brought into main. Noticed by @rhine3

The bug can be seen here:
https://github.com/RolnickLab/ami-data-companion/blob/main/trapdata/api/models/classification.py#L78

One fix can be found here:
https://github.com/RolnickLab/ami-data-companion/blob/6a1b16fe5f03efdfc644e096a436acdb90f7c37b/trapdata/api/models/classification.py#L73-L128
